requirements.in

```diff
@@ -0,0 +1,68 @@
+# tensor search:
+requests==2.28.1
+anyio==3.7.1
+fastapi==0.86.0
+uvicorn==0.31.0
+fastapi-utils==0.2.1
+jsonschema==4.17.1
+redis==4.4.2
+
+# s2_inference:
+more_itertools==10.4.0
+boto3==1.25.4
+botocore==1.28.4
+nltk==3.9.1
+Pillow==10.4.0
+numpy==1.23.4
+validators==0.20.0
+sentence-transformers==2.2.2
+open_clip_torch==2.24.0
+clip-marqo==1.0.2
+protobuf==3.20.1
+onnx==1.12.0
+onnxruntime==1.13.1
```

Comment:
Do we need both this and the gpu version on amd64? Also should we use matching version for cpu and gpu package?

Reply:
Yes, we need both of them since we don't have a specific CPU image. They are compatible based on https://onnxruntime.ai/docs/reference/compatibility.html